src/mcts/search.cc

```diff
@@ -183,14 +183,23 @@ int64_t Search::GetTimeToDeadline() const {
       .count();
 }
 
+namespace {
+
+inline float ComputeCpuct(const SearchParams& params, uint32_t N) {
+  const float init = params.GetCpuct();
+  const float k = params.GetCpuctFactor();
+  const float base = params.GetCpuctBase();
+  return init + (k ? k * std::log((N + base) / base) : 0.0f);
```

Comment:
As borg mentioned, deepmind version has a +1 in here. (I really don't think its significant unless someone tries to test very small 'base' parameters.)

Reply:
Yeah I didn't understand that 1+ so decided to simplify.